src/compute/src/sink/correction.rs

```diff
@@ -161,26 +161,27 @@ impl<D> CorrectionV1<D> {
 
 impl<D: Data> CorrectionV1<D> {
     /// Insert a batch of updates.
-    pub fn insert(&mut self, mut updates: Vec<(D, Timestamp, Diff)>) {
+    pub fn insert(&mut self, updates: &mut Vec<(D, Timestamp, Diff)>) {
         let Some(since_ts) = self.since.as_option() else {
             // If the since frontier is empty, discard all updates.
             return;
         };
 
-        for (_, time, _) in &mut updates {
+        for (_, time, _) in &mut *updates {
```

Comment:
Isn't this equivalent to just using `updates` directly?

Reply:
For some reason it's not. If you try that you get an error with the helpful hint that you should use a reborrow instead 🤷